packages/mux-uploader/src/mux-uploader-status.ts

```diff
@@ -0,0 +1,76 @@
+import { globalThis, document } from 'shared-polyfills';
+import { getMuxUploaderEl } from './utils/element-utils';
+import { type MuxUploaderElementEventMap } from './mux-uploader';
+import type MuxUploaderElement from './mux-uploader';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+<style>
+
+:host([upload-error]) {
+  color: #e22c3e;
+}
+</style>
+
+<span id="status-message" role="status" aria-live="polite"></span>
+`;
+
+class MuxUploaderStatusElement extends globalThis.HTMLElement {
+  statusMessage: HTMLElement | null | undefined;
```

Comment:
suggestion(non-blocking): We should probably be consistent with:

1. naming (`#uploaderEl` vs. `statusMessage`)
2. access scope (`#uploaderEl` is private, `statusMessage` is public)

Given the impact on e.g. tests, I'd recommend treating this as a followup.